src/index.tsx

```diff
@@ -172,16 +172,31 @@ function useWavesurferEvents(wavesurfer: WaveSurfer | null, events: OnWavesurfer
   }, [wavesurfer, ...flatEvents])
 }
 
+function useWavesurferLoad(
+  wavesurfer: WaveSurfer | null,
+  url: string | undefined,
+  peaks: (Float32Array | number[])[] | undefined,
+  duration: number | undefined,
+) {
+  useEffect(() => {
+    if (!wavesurfer) return
+    if (!url) return
```

Comment:
URL can be empty, it should still render a silent waveform.

Reply:
Thank you. I'll make the update soon and re-push.